fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/DefaultOCFLObjectSession.java

```diff
@@ -47,11 +46,12 @@
 import java.util.function.Consumer;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
+
+import edu.wisc.library.ocfl.api.model.VersionId;
```

Comment:
Very minor... but it seems that this `import` statement should be together with the above `edu.wisc` imports.

Reply:
I'm usually an agrressive import organizer, but I've been fighting hard to resist my urge to organize them because my IDE (intellij) organizes them differently than they are currently (I assume eclipse's work). If people don't care about resides import reorgs, I'll fire away.

Reply:
organized imports